firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -848,7 +856,30 @@ class SearchViewController: SiteTableViewController,
             break
         }
     }
-}
+
+    private func filterData(data: Cursor<Site>) -> [Site?] {
```

Comment:
Hmm, the `[Site?]` is a little unfortunate—can we make it `[Site]` and drop the optionals?